src/components/resume-builder/components/edit-panel/SocialHandlesEditBox.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useMemo } from 'react';
+
+// THIRD_PARTY
+import {
+  DragDropContext,
+  Droppable,
+  Draggable,
+  type DropResult,
+} from '@hello-pangea/dnd';
+
+// HOOKS
+import { useResumeData } from '../../context/ResumeDataContext';
+
+// COMPONENTS
+import { InputField } from './InputField';
+import { DraggableWrapper } from '../wrappers/DraggableWrapper';
+import { BlueButton, ButtonWithCrossIcon } from './EditPanelComponents';
+
+// TYPES
+import type { SocialHandle } from '../../types/resume-data';
+
+type SocialHandleKeys = 'label' | 'link';
+
+export const SocialHandlesEditBox: React.FC = () => {
+  const { socialHandles, updateSocialHandles } = useResumeData();
+
+  const onDragEnd = (result: DropResult<string>) => {
```

Comment:
Understand this logic, as this might be asked by someone who is reading the code. Comment the understanding.